server.py

```diff
@@ -8,8 +9,13 @@ def desk():
     return render_template("index.html")
 
 
+@app.route("/word/recognize/")
+def req():
+    data = request.args.get('word', {}, type=str)
+    return jsonify(dict(results=str(data)))
+
 # def ask(word):#
 
 
 if __name__ == "__main__":
-    app.run()
+    app.run(host='0.0.0.0')

```

Comment:
Why have you changed `run()` to `run(host=...)` even though I advised you not to use this in the previous PR? Is there some problems with runtime or what?